Added Ansible playbook for uploading multiple keys to authorized_keys…

StaffanArvidsson opened the pull request:
… for the master

This playbook will add keys from the directory: 

> [KubeNowDir]/public_keys/ 

Into the masters Aurthorized_keys. Another possibility is to make this as the default and make it possible to give a user-defined directory as a parameter.

mcapuccini commented:
I need to understand how to change branch name on a PR, so the tests will run. I'll merge as soon as I'll manage!

mcapuccini commented:
@StaffanArvidsson didn't you try `{{ (public_keys_dir | default(playbook_dir ~ '/..' )) ~ '/*.pub' }}`?

StaffanArvidsson commented:
@mcapuccini no I did not have time to try using the variable but I will do it now and hopefully push an update tonight. Also I will add a `public_keys` directory with a README.md file

StaffanArvidsson commented:
@mcapuccini @andersla I added the possibility of setting a user-defined directory instead of being forced to put the keys into the specific directory. Also added a README file for the `public_keys` directory explaining the basics of the usage of the playbook (which should be useful for writing things in the docs at later stage).

mcapuccini commented:
@StaffanArvidsson could you add documentation here: https://github.com/kubenow/docs. It would be good if you can create a new section "Operations" on the right menu and add this new feature there.

StaffanArvidsson commented:
@mcapuccini I will probably not come around fixing that until I'm back at work after 20th of April. I thought you were going to redo the Ansible part and did not know where I should put documentation about it? Should it be put in a completely new page?

mcapuccini commented:
@StaffanArvidsson yep, it should be a new page. I have nothing to put in apart from this ansible playbook, so this would be a good chance for creating this section. It's fine if you do this after the 20th, no hurry on our side :) Does @jonalv need it before? @jonalv could you take care of the docs part?

jonalv commented:
Well we don't really need this until @StaffanArvidsson  is back because the feature is for allowing both of us to access and if it's only me then it's hardly needed :)

But maybe I will find some round tu-its and do it. Who knows? :)

mcapuccini commented:
Closing for inactivity
